ui/DesignSystem/Component/Sidebar.svelte

```diff
@@ -124,56 +118,20 @@
         </a>
       </Tooltip>
     </li>
-
-    {#each orgs as org}
+  </ul>
+  <ul class="bottom">
+    {#if isDev()}
```

Comment:
We use this check here before. Wouldn't introduce it in this PR.

Reply:
I thought we didn't want to show unfinished features, is that not the case?